### PR TITLE
feat(app): GitHub integration

### PR DESCRIPTION
**Pull request by HonzaTuron**
Adding simple walkthrough for new GitHub integration

### CODE REVIEW

**Review by jbartadev — CHANGES_REQUESTED** [review]
1. I think we should also add the other step with the list of repositories - we can use the Apify account for the screenshot and maybe display a couple of our repos to explain this better with all the options (public projects as `crawlee` or `got-scraping` and also some private repo)
Snímek obrazovky 2023-06-02 v 9 54 42
2. it'd be nice to describe the options:

- Searching through repositories
- Adding different account
- Managing permissions

File: sources/platform/integrations/github.md [hunk 1]
```diff
@@ -20,7 +20,28 @@ To use the Apify integration for GitHub, you will need:
 
 ### Create an Actor from a GitHub repository
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NEzT_p_RE1Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+Learn how to create an Actor from a GitHub repository. This is useful if you want to use the GitHub integration to automatically deploy your code to Apify.
```
Comment (jbartadev) [inline sources/platform/integrations/github.md:23]:
```suggestion
Learn how to create an Actor from a GitHub repository. This is useful if you want to automatically deploy and build your code on push to your GitHub repository.
```

File: sources/platform/integrations/github.md [hunk 2]
```diff
@@ -20,7 +20,28 @@ To use the Apify integration for GitHub, you will need:
...
+Learn how to create an Actor from a GitHub repository. This is useful if you want to automatically deploy and build your code on push to your GitHub repository.
+
+![New actor page](./images/apify-git-repository.png)
+
+Selecting _Link Git repository_ will open a new modal to select a provider to use.
+Selecting _GitHub_ will open a new window with GitHub authentication and selecting GitHub repository.
+
+![Git provider select modal](./images/apify-git-provider-select.png)
+
+To link account, click on _Add GitHub account_ and follow instructions on github.com. Certain organization or user can be selected.
```
Comment (TheoVasilis) [inline sources/platform/integrations/github.md:32]:
```suggestion
To link an account, click on _Add GitHub account_ and follow the instructions on github.com. Certain organizations or users can be selected.
```

File: sources/platform/integrations/github.md [hunk 3]
```diff
@@ -20,7 +20,28 @@ To use the Apify integration for GitHub, you will need:
...
+
+![Git repository account select](./images/apify-git-repository-add.png)
+
+You can switch among all authorized users and organizations.
+
+![Git repository account select](./images/apify-git-repository-account.png)
+
+If required repository is missing, try to find it with _Search_.
```
Comment (TheoVasilis) [inline sources/platform/integrations/github.md:40]:
```suggestion
If the required repository is missing, try finding it with _Search_.
```

File: sources/platform/integrations/github.md [hunk 4]
```diff
@@ -20,7 +20,28 @@ To use the Apify integration for GitHub, you will need:
...
+
+![apify-git-repository-search.png](./images/apify-git-repository-search.png)
+
+Actor is created immediately from selected repository.
```
Comment (TheoVasilis) [inline sources/platform/integrations/github.md:44]:
```suggestion
An Actor is created immediately from the selected repository.
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

**Review by TheoVasilis — APPROVED** [review]

**Review by jbartadev — APPROVED** [review]

**Comment by jbartadev** [comment]
@HonzaTuron let's merge this as the GH integration is already out!

**Pull request merged**